### PR TITLE
Display CarbonFootprint in CardView and add Environment Tab

### PR DESCRIPTION
**Pull request by KarthikRIyer**
## Description

Carbon Footprint now displayed inside a CardView. 
Also added an Environment Tab to which Recycling Instructions will be added later.
 
whatsapp image 2019-01-08 at 2 41 58 am
whatsapp image 2019-01-08 at 2 41 38 am

### CODE REVIEW

**Comment by teolemon** [comment]
Can you move the Carbon footprint to the environment tab and hide the tab if it's not present?

**Comment by PrajwalM2212** [comment]
@teolemon Is the environment tab only for OFF or even for OPF,OPFF and OBF ?

**Comment by teolemon** [comment]
I would conditionnally show it on Open Food Facts, if and only if we have a carbon footprint.
Once we merge the PR for recycling instructions, I want to move recycling instructions and packaging in that tab.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java [hunk 1]
```diff
@@ -264,6 +265,7 @@ private void setupViewPager( ViewPager viewPager )
 		{
 			adapterResult.addFragment( new NutritionProductFragment(), menuTitles[2] );
 			adapterResult.addFragment( new NutritionInfoProductFragment(), menuTitles[3] );
+			adapterResult.addFragment( new EnvironmentProductFragment(), "Environment" );
```
Comment (PrajwalM2212) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java]: If environment tab is OFF only the a guard has to be put here.

Comment (PrajwalM2212) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java]: You need to add this to `menuTitles` string array and then reference it here. Do the same for other entries that may have missed this logic.

Comment (KarthikRIyer) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java]: Do I need to add it manually to all the languages? Adding it to the default strings.xml doesn't seem to work

Comment (huzaifaiftikhar) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/ProductActivity.java]: @KarthikRIyer No, you don't need to add it manually to all the languages, just add the String to the default `string.xml` file. The translations are handled by Crowdin. You don't need to worry about that.

**Comment by teolemon** [comment]
Thank you @KarthikRIyer 👍 
Thanks @PrajwalM2212 for the review and feedback

**Pull request merged**